src/Maestro/tests/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -272,5 +297,49 @@ public async Task CheckoutRemoteRefAsync(string commit)
             await RunGitAsync("fetch", "origin", commit);
             await RunGitAsync("checkout", commit);
         }
+
+        internal IImmutableList<AssetData> GetAssetData(string asset1Name, string asset1Version, string asset2Name, string asset2Version)
+        {
+            ImmutableList<AssetData> sourceAssets = ImmutableList<AssetData>.Empty;
+            AssetData asset1 = new AssetData(false)
+            {
+                Name = asset1Name,
+                Version = asset1Version,
+                Locations = ImmutableList<AssetLocationData>.Empty
+            };
+            asset1.Locations.Add(new AssetLocationData(LocationType.NugetFeed)
+            { Location = @"https://pkgs.dev.azure.com/dnceng/public/_packaging/NotARealFeed/nuget/v3/index.json" });
+            sourceAssets = sourceAssets.Add(asset1);
+
+            AssetData asset2 = new AssetData(false)
+            {
+                Name = asset2Name,
+                Version = asset2Version,
+                Locations = ImmutableList<AssetLocationData>.Empty
+            };
+            asset2.Locations.Add(new AssetLocationData(LocationType.NugetFeed)
+            { Location = @"https://pkgs.dev.azure.com/dnceng/public/_packaging/NotARealFeed/nuget/v3/index.json" });
+            sourceAssets = sourceAssets.Add(asset2);
+
+            return sourceAssets;
+        }
+
+        public async Task SetRepositoryPolicies(string repoUri, string branchName, string[] policyParams = null)
+        {
+            policyParams = policyParams?? new string[]{""};
+            string[] commandParams = { "set-repository-policies", "-q", "--repo", repoUri, "--branch", branchName };
+
+            if (policyParams != null)
```

Comment:
This if can't not trigger, because of line 329.  I think line 329 probably isn't necessary.